modules/appeals_api/spec/services/appeals_api/evidence_submission_request_validator_spec.rb

```diff
@@ -9,7 +9,7 @@ module AppealsApi
       context 'Notice of Disagreement' do
         it 'returns an error if the NOD does not exist' do
           result = described_class.new(
-            'fake_id', 'irrelevant_ssn', 'NoticeOfDisagreement'
+            'fake_id', 'irrelevant_file_number', 'NoticeOfDisagreement'
```

Comment:
I appreciate testing the change from SSN to SSN or FileNumber, but also moving all the created NODs to v2 seems premature at the moment.

Reply:
Ok. I figured since the ssn is tested in SC, that I'd just test file number, but I can return some of them to v1